docs/tutorials/sampler.ipynb

```diff
@@ -0,0 +1,211 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    print(bell.draw())
</pre><p>This will not work well if the drawing default option is set to <code>mpl</code>. I suggest changing to <code>print(bell.draw(output='text'))</code> or <code>bell.draw()</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>